packages/base/components/Image/ImageWithAttributionContainer.vue

```diff
@@ -0,0 +1,109 @@
+<script setup>
+defineProps({
```

Comment:
This looks like prop drilling as most of these are just passed on down to ImageWithAttribution.

Could the styling be moved down to ImageWithAttribution, and either always applied or made conditional, e.g. with an `asContainer` prop?

Or should we make rendering the ImageWithAttribution itself the parent's responsibility via a slot?
```vue
<ImageWithAttributionContainer>
  <ImageWithAttribution
    :src="src"
  />
</ImageWithAttributionContainer>
```